hash/hash.go

```diff
@@ -1,6 +1,6 @@
 package hash
 
-import "github.com/spacemeshos/sha256-simd"
+import "github.com/minio/sha256-simd"
```

Comment:
why this update?

Reply:
This seems to be the only repo where we are still using our (outdated) fork over the upstream version if the library.

Reply:
was the test failure caused by minio?

Reply:
Unfortunately not, it seems like golang.org/x/net is the culprit :-( at least if I update the dependency tests start failing. I will still put the migration from `github.com/spacemeshos/sha256-simd` to `github.com/minio/sha256-simd` into its own PR.